test/test_transactions.py

```diff
@@ -389,7 +392,7 @@ def test_transaction_direct_connection(self):
             with client.start_session() as s, s.start_transaction():
                 res = f(*args, session=s)  # type:ignore[operator]
                 if isinstance(res, (CommandCursor, Cursor)):
-                    list(res)
+                    res.to_list()
```

Comment:
Can you add a docstring and changelog entry for to_list?

Reply:
`list(cursor)` still works on synchronous cursors, but asynchronous cursors require the use of `Cursor.to_list()` instead. This change is purely so our tests can be synchronized smoothly.

Reply:
Right, the only issue is that to_list() is undocumented.